Drivers/MotorChannel/Inc/ZP_D_PWMChannel.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef ZP_D_PWM_CHANNEL_HPP_
+#define ZP_D_PWM_CHANNEL_HPP_
+
+#include "main.h"
+
+#include "ZP_D_MotorChannel.hpp"
+
+class PWMChannel : public MotorChannel {
+    public:
+        PWMChannel(TIM_HandleTypeDef* timer,
+					uint16_t timerChannel);
+
+        void set(uint8_t percent);
+    private:
+    	// set to minimum and max amount of counts in a duty cycle
+        uint32_t minSignal;
+        uint32_t maxSignal;
+        uint32_t PERIOD_TICKS;
```

Comment:
These variables aren't mutated by the class so we can mark them as `const` and bring back the all caps naming.
Additionally, `PERIOD_TICKS` doesn't need to be saved by the class because it isn't used, except in the constructor.
```suggestion
        const uint32_t MIN_SIGNAL;
        const uint32_t MAX_SIGNAL;
```